ds701_book/20-Recommender-Systems.qmd

```diff
@@ -23,27 +22,28 @@ import laUtilities as ut
 %matplotlib inline
 ```
 
-
-
 Today, we look at a topic that has become enormously important in society: recommender systems.
 
 We will
+
 * Define recommender systems
 * Review the challenges they pose
 * Discuss two classic methods:
     * Collaborative Filtering
     * Matrix Factorization
 
-```{margin}
-This section draws heavily on 
+::: aside
+This section draws heavily on
+
 * These [slides](http://alex.smola.org/teaching/berkeley2012/slides/8_Recommender.pdf) by Alex Smola
 * [Matrix Factorization Techniques for Recommender Systems,](https://ieeexplore.ieee.org/document/5197422) by Yehuda Koren, Robert Bell, and Chris Volinsky, and
 * [Collaborative Filtering with Temporal Dynamics,](https://dl.acm.org/doi/10.1145/1557019.1557072) by Yehuda Koren
-```
+:::
 
 ## What are Recommender Systems?
 
 The concept of recommender systems emerged in the late 1990s / early 2000s as social life moved online:
+
 * online purchasing and commerce
 * online discussions and ratings
 * social information sharing
```

Comment:
In the sentence below can you change it to:

The amount of content was exploding...